dnsapi/dns_one.sh

```diff
@@ -41,11 +57,15 @@ dns_one_add() {
   postdata="$postdata&password1=$ONECOM_Password"
   postdata="$postdata&loginTarget="
   #_debug postdata "$postdata"
-
+  
+  #CURL does not work
+  local tmp_USE_WGET=$ACME_USE_WGET
+  ACME_USE_WGET=1
```

Comment:
No,  never select to use wget or curl in your script.
It's control by acme.sh

if curl does not work,  you can display an `_err` message to ask the user to use wget wih `--use-wget`

Reply:
I still don't understand why curl is not working ?

Reply:
the response from curl is every time the login page.
the same results on other devices.
i will try to get more info

Reply:
Without the follow redirects (Option) "-L" it works with curl

Reply:
i fixed to work with curl